src/components/table/Table.vue

```diff
@@ -1169,7 +1169,8 @@ export default {
                         if (value !== Number(input)) return false
                     } else {
                         const re = new RegExp(escapeRegExpChars(input), 'i')
-                        if (!re.test(value)) return false
+                        const valueWithoutDiacritics = removeDiacriticsFromString(value)
+                        if (!re.test(valueWithoutDiacritics) && !re.test(value)) return false
```

Comment:
It keeps filtering if I search using accents or not. 
I thought to create a new param in the column to enable the filter with accents, but not sure if it is necessary. 
What do you think about it?

Reply:
`OR` could be enough here?

```suggestion
        return !re.test(valueWithoutDiacritics) || !re.test(value)
```

Reply:
Nice catch!
I just need to remove the `!` in the condition and it worked.

Reply:
Cool!